src/traces/scattermapbox/attributes.js

```diff
@@ -18,6 +18,7 @@ var colorScaleAttrs = require('../../components/colorscale/attributes');
 
 var extendFlat = require('../../lib/extend').extendFlat;
 var overrideAll = require('../../plot_api/edit_types').overrideAll;
+var mapboxLayoutAtributes = require('../../plots/mapbox/layout_attributes')
```

Comment:
Missing a semicolon at the end of this line.
```
npm run test-syntax
```
could also catch these.

Reply:
Thanks